src/timearray.jl

```diff
@@ -56,6 +56,11 @@ convert(::Type{TimeArray{Float64, 2}}, x::TimeArray{Bool, 2}) =
 convert(x::TimeArray{Bool, 1}) = convert(TimeArray{Float64, 1}, x::TimeArray{Bool, 1})
 convert(x::TimeArray{Bool, 2}) = convert(TimeArray{Float64, 2}, x::TimeArray{Bool, 2})
 
+###### copy ###############
+
+copy(ta::TimeArray)::TimeArray =
```

Comment:
I think our compiler is smart enough, and we do not need return type annotation here.

Reply:
Even if it is redundant, I prefer to annotate types as much as possible. But I will adapt to the style you prefer. Would you prefer to remove the output type?

Reply:
ok, I'm fine with current.